ios/FluentUI/Badge Field/BadgeViewTokenSet.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined styles of the Badge.
+@objc(MSFBadgeViewStyle)
+public enum Style: Int {
+    case `default`
+    case danger
+    case severeWarning
+    case warning
+    case success
+    case neutral
+}
+
+/// Pre-defined sizes of the Badge.
+@objc(MSFBadgeViewSize)
+public enum Size: Int, CaseIterable {
```

Comment:
Same comment for `Size` here as above for `Style` - this naming is not scoped to `BadgeView`, but it seems like it should be.

Reply:
Also we've moved away from `Size` and are now preferring to use `SizeCategory`, to differentiate from the traditional `CGSize` structs.